mods/ra/weapons/missiles.yaml

```diff
@@ -448,6 +308,7 @@ SubMissileAA:
 		ValidTargets: Air, Ground, Water
 
 SCUD:
+	Inherits: ^AntiGroundMissile
```

Comment:
This changes the `HorizontalRateOfTurn`, `Arm`, `RangeLimit`, and `ContrailLength`.  Are you *really* sure about this?